OmniKit/PumpManager/OmnipodPumpManager.swift

```diff
@@ -1378,18 +1378,28 @@ extension OmnipodPumpManager: PumpManager {
                 state.bolusEngageState = .engaging
             })
 
-            // If pod suspended, resume basal before bolusing to match existing Medtronic PumpManager behavior
+            // Initialize to true to match existing Medtronic PumpManager behavior for any
+            // manual boluses or to false to never auto resume a suspended pod for any bolus.
+            let autoResumeOnManualBolus = true
```

Comment:
nit: please don't add static configuration variables. makes line 1387 more complicated to parse, and if this is a decision we want to make in the code, it should be made authoritatively.

Reply:
Most parents I have discussed the auto resume on bolus feature with would prefer to not have this potentially risky behavior. But since I was told to maintain this behavior for Omnipod, I wanted to document this non-obvious behavior in a way which allows for its convenient disabling.